templates/node/README.md

```diff
@@ -17,7 +16,7 @@ The template is responsible for managing a virtual machine(VM) on an openVCloud
 - `dataDiskMountpoint`: default: `/var`.
 - `dataDiskFilesystem`: type of filesystem
 - `description`: arbitrary description of the VM. **optional**
-- `ports`: list of port forwards of the VM. Ports can be configured during installation of the vm or with actions `['portforward_create']`, `['portforward_delete']`.
+- `ports`: list of port forwards of the VM. Ports can be configured with actions `['portforward_create']`, `['portforward_delete']`.
```

Comment:
please specify that the ports can be "only" configured through actions .. and will be filled automatically .. I'm assuming what i'm saying is right :)